cmd/nodecmd/local.go

```diff
@@ -435,13 +437,13 @@ func localValidate(_ *cobra.Command, args []string) error {
 		}
 	}
 
-	if rpcURL == "" {
-		rpcURL, err = app.Prompt.CaptureURL("What is the RPC endpoint?", false)
+	if localValidateFlags.RPC == "" {
+		localValidateFlags.RPC, err = app.Prompt.CaptureURL("What is the RPC endpoint?", false)
```

Comment:
Since in RPC flag validation we use command specific logic, I think this validation should be done there (like we do for `add_validator`